apps/api/.env.example

```
@@ -30,3 +30,6 @@ ZEPTOMAIL_TOKEN=zeptomail-token
 # key for the encryption
 # can be created by running this: echo "$(openssl rand -hex 32)opensox$(openssl rand -hex 16)"
 ENCRYPTION_KEY=encryption-key
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)